azavea/tilejson.io
# Add Netlify support

## Overview

Add support for Netlify based deployments. In the process, replace the Docker based Node.js development environment with one that is native to the Vagrant virtual machine OS (Ubuntu 16.04).

### Demo

See: https://5bf385b7e4708573ceabafe4--tilejson.netlify.com/

### Notes

The hope is that this PR can set the stage for testing Netlify's built-in support for OAuth Providers (like GitHub) via https://www.netlify.com/docs/authentication-providers/.

## Testing Instructions

- Destroy your development environment
- Bring up the development environment using the instructions in the `README`
- Ensure that `cibuild` produces a legitimate Webpack bundle
- Ensure that the Netlify PR preview appears as expected
